backend/package.json

```
@@ -1,21 +1,34 @@
 {
   "name": "backend",
-  "version": "0.1.0",
-  "private": true,
   "scripts": {
-    "dev": "cargo run",
-    "build": "cargo build && bun run generate",
-    "generate": "concurrently --kill-others-on-fail --prefix-colors=gray,blue bun:generate:*",
-    "generate:json": "cargo run --bin generate && prettier --write --semi=false ../generated/openapi/schema.json",
+    "dev": "tsx watch src/index.ts",
+    "build": "tsx build ./build.ts",
+    "generate": "tsx src/bin/genereate.ts",
     "generate:typescript": "openapi-typescript ../generated/openapi/schema.json -o ../generated/openapi/schema.ts && prettier --write --semi=false ../generated/openapi/schema.ts",
```

`backend-ci`のgenerateジョブでは`bun run generate`しか実行していないので、`bun run generate`でopenapi.jsonと型定義の両方が生成されるようにする必要があると思われます！

```suggestion
    "generate": "bun run generate && bun run generate:typescript",
    "generate:json": "tsx src/bin/genereate.ts",
    "generate:typescript": "openapi-typescript ../generated/openapi/schema.json -o ../generated/openapi/schema.ts && prettier --write --semi=false ../generated/openapi/schema.ts",
```